html/janus.js

```diff
@@ -2301,46 +2306,38 @@ function Janus(gatewayCallbacks) {
 						videoSupport = media.video;
 					} else {
 						var width = 0;
-						var height = 0, maxHeight = 0;
```

Comment:
why `maxHeight` has been deleted ?

Reply:
`maxHeight` does not appear to be used by anything.

Reply:
in the form maxHeight it's for old Chromia, but note that you can still pass min and max (which doesn't make much sense if they're exactly the same, of course)

https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia